src/main/resources/META-INF/resources/primefaces/datatable/datatable.css

```diff
@@ -60,6 +60,7 @@
 .ui-datatable .ui-column-filter {
     display: block;
     width:100px;
```

Comment:
This can cause problem if we have an extra large column

for example:

![image](https://user-images.githubusercontent.com/10467831/31837734-816aaf1c-b59f-11e7-8d3d-70628053ea59.png)

Reply:
I expect it to almost fill the column by default. What do you think ?